src/ec/.storybook/preview-head.html

```diff
@@ -25,11 +30,50 @@
 ></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.8.3/polyfill.min.js"></script>
 <script src="https://unpkg.com/svg4everybody@2.1.9/dist/svg4everybody.min.js"></script>
+<script src="https://unpkg.com/@popperjs/core@2"></script>
+<script src="https://unpkg.com/tippy.js@6"></script>
```

Comment:
I like the new way of visualising internal errors with the tooltips, however, since the components are destined to be used outside of storybook, how will this react in a third-party application? like before?

Reply:
Little idea when checking the PR .. would be to have a system like drupal theme-debug, ie that most of the issues or messages becomes html comments in code via inspect .. a visual icon remain to inspect them..

Reply:
So, the inner_check uses a standard "title" attribute. This fancy tooltips are only for ecl-twig, to improve the demo, but it would work in any environment, just using the standard overlay handled by the browser.
I'm not sure i get the idea in the second part of your sentence.